app-tracking-protection/vpn-impl/src/main/res/layout/include_trackers_toolbar.xml

```diff
@@ -28,7 +28,7 @@
         android:id="@+id/trackers_toolbar"
         android:layout_width="match_parent"
         android:layout_height="?attr/actionBarSize"
-        android:background="?attr/daxColorSurface"
+        android:background="?attr/daxColorToolbar"
```

Comment:
we need to create a new attribute for the toolbar color. the new theme uses background color instead of surface.